packages/analytics-js-integrations/src/integrations/BingAds/index.js

```diff
@@ -1 +1 @@
-export { BingAds } from './browser';
+export { default as BingAds } from './browser';
```

Comment:
```suggestion
export BingAds from './browser';
```

Reply:
@koladilip this will not work as we are exporting default from browser. in `index.js` we need to export it as default only